jekyll/_cci2/prevent-unregistered-users-from-spending-credits.adoc

```diff
@@ -0,0 +1,43 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Prevent unregistered users from spending credits
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A how to guide to configuring your org so that unregistered users will not be able to perform actions that spend credits3
+:icons: font
+:experimental:
+
+It is possible for _unregistered_ users to trigger pipelines, and therefore, spend credits. Use the **Prevent unregistered user spend** option to only allow registered users the ability to trigger work that consumes credits. For a full breakdown of the various ways pipelines can be triggered, see the xref:triggers-overview[Triggers overview].
+
+[NOTE]
+====
+An _unregistered_ user is how CircleCI tracks users who have triggered builds but have not signed up for a CircleCI account. Unregistered users **do** count as link:https://support.circleci.com/hc/en-us/articles/360034783154[active users].
```

Comment:
It's described in the next linked document (https://support.circleci.com/hc/en-us/articles/360037744473-What-is-an-Unregistered-User-) but Unregistered users will map to a single "Unregistered User", though I'm not sure if we need to make that more clear here.

Reply:
@BrennanMacKay so multiple unregistered for an org would just be counted as a single active user?

Reply:
Yep that's correct, I'm not sure exactly where that mapping occurs but we have a single unregistered user actor id that all of the unregistered users are mapped to. This is what shows up as the `Unregistered User` in the user list.